src/main/java/seedu/recruittrackpro/logic/commands/FindCommand.java

```diff
@@ -13,7 +16,7 @@
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
```

Comment:
Update JavaDocs as per this class intention, consider the following:
```suggestion
 * Finds and lists all persons in RecruitTrackPro whose names or attributes contain any of the argument keywords.